sky/serve/replica_managers.py

```diff
@@ -567,7 +570,8 @@ def __init__(self, service_name: str,
         self._update_mode = serve_utils.DEFAULT_UPDATE_MODE
         logger.info(f'Readiness probe path: {spec.readiness_path}\n'
                     f'Initial delay seconds: {spec.initial_delay_seconds}\n'
-                    f'Post data: {spec.post_data}')
+                    f'Post data: {spec.post_data}\n'
+                    f'Readiness headers: {spec.readiness_headers}')
```

Comment:
Seems we are still printing out the headers? Should we use `list(spec.readiness_headers.keys())` instead?

Reply:
Good catch! Fixed. Thanks!